(DOCSP-33221): "Release notes, redirects, source constant updates for v1.11.0 and v1.12.0. (#203)
mongodb/docs-atlas-cli: source/atlas-cli-changelog.txt

   :depth: 1
   :class: singlecol

.. _atlas-cli-1.12.0:

{+atlas-cli+} 1.12.0
--------------------

> - Drops support for Ubuntu 18.04.

Reviewer: Suggestion: With such big announcements this release, maybe make this last?
```suggestion
- Drops support for Ubuntu 18.04.
```

- Adds :ref:`atlas-deployments` commands so that you can work with
  |service|, including using |fts| and {+avs+}, throughout the entire 

>   software development lifecycle, from your local environment to the 

Reviewer: ```suggestion
  software development lifecycle from your local environment to the 
```

  cloud. To learn more, see :ref:`atlas-cli-local-cloud`.

> - Adds the following new commands to manage {+atlas-sp+} instances

Reviewer: Suggestion: link to the Streams Processing page for more info if they're unfamiliar with it

>   and connection registries:

Reviewer: Question: are these the first streams processing commands or were there other ones released before this? If they're the first consider just linking to the top-level streams processing command to align with what you did with atlas deployments (or, conversely, list out the deployments commands). This is optional though, not blocking

  - :ref:`atlas-streams-instances-create`
  - :ref:`atlas-streams-instances-delete`
  - :ref:`atlas-streams-instances-describe`
  - :ref:`atlas-streams-instances-list`
  - :ref:`atlas-streams-instances-update`
  - :ref:`atlas-streams-connections-create`
  - :ref:`atlas-streams-connections-delete`
  - :ref:`atlas-streams-connections-describe`
  - :ref:`atlas-streams-connections-list`
  - :ref:`atlas-streams-connections-update`

{+atlas-cli+} 1.11.0
--------------------

- Fixes an issue that occurred when you created x509 certs for database 
  users. The command didn't output the certificate. If you depend on 
  the previously incorrect output, this update is a breaking change.

- :ref:`atlas-alerts-settings` create and update commands perform 

>   validation that helps you formulate all required arguments per each 

Reviewer: Nit: "for each" is more commonly used than "per each" so will improve reader comprehension
```suggestion
  validation that helps you formulate all required arguments for each 
```

  alert type.

- Fixes a bug where the :ref:`atlas-projects-list` command ignored the 
  ``--orgId`` flag.

> - Fixes an issue with running cloud provider access commands.

Reviewer: ```suggestion
- Fixes an issue that occurred while running cloud provider access commands.
```

> - Supports configuring alerts with a file.

Reviewer: ```suggestion
- Supports configuring alerts with a file. To learn more,
  see :doc:`</reference/json/alert-config-file>.
```
